packages/widgetbook_models/README.md

```diff
@@ -0,0 +1,87 @@
+<img height=40 src="https://media.githubusercontent.com/media/widgetbook/widgetbook/main/docs/assets/Widgetbook_Logo.png">
+
+[![Discord](https://img.shields.io/discord/879618555560218625?color=blue&style=flat-square)](https://discord.com/invite/zT4AMStAJA)
+[![style: very good analysis](https://img.shields.io/badge/style-very_good_analysis-B22C89.svg?style=flat-square)](https://pub.dev/packages/very_good_analysis) 
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?style=flat-square)
+![GitHub Workflow Status](https://img.shields.io/github/workflow/status/widgetbook/widgetbook/ci?label=test&style=flat-square)
+
+This package features the models for the Widgetbook `Device` class. The `device` class is used in the following packages:
+
+- [package:widgetbook](https://pub.dev/packages/widgetbook)
+- [package:widgetbook_annotation](https://pub.dev/packages/widgetbook_annotation)
+- [package:widgetbook_generator](https://pub.dev/packages/widgetbook_generator)
+
+Furthermore, the package contains predefined and commonly used devices.
+
+> **NOTE**: It is not required to add this package to your project. [package:widgetbook_models](https://pub.dev/packages/widgetbook_models) was designed to make the classes defined accessible to the other packages. [package:widgetbook](https://pub.dev/packages/widgetbook) exports the classes defined in the package so you can use them without needing to import anything else than [package:widgetbook](https://pub.dev/packages/widgetbook). 
+
+# Classes
+
+Currently the package contains the `Device` class with its required subsclasses and collections of various devices.
+
+## Resolution
+
+A device's screen resolution is defined by the number of distinct pixels in each dimension that can be displayed. Furthermore, on Apple and Android devices, this resolution is divided into logical and native resolution by a scaling factor. Therefore, to define a resolution, the native amount of pixels in each dimension and a scaling factor is required. 
+
+A `Resolution` instance is created by using
+
+```dart
+Resolution(
+    nativeSize: DeviceSize(width: 1668, height: 2388),
+    scaleFactor: 2,
+),
+```
+
+`DeviceSize` is similar to `Size` from the Flutter SDK and contains properties for `width` and `height`.
+
+## Device
+
+The device class specifies a device like the *iPhone 12* based on a `name`, its `resolution` and its `type`. `type` is primarily used for better iconization in [package:widgetbook](https://pub.dev/packages/widgetbook)'s user interface.
+
+An Apple iPhone 12 can be defined by using
+
+```dart
+Device iPhone12 = Device(
+    name: 'iPhone 12',
+    resolution: Resolution(
+        nativeSize: DeviceSize(width: 1170, height: 2532),
+        scaleFactor: 3,
+    ),
+    type, DeviceType.mobile,
+);
+```
+
+### Device Types
+
+The following device types exist and can be accessed with the respective `Device` constructor:
+
+- `Device.watch( ... )`
```

Comment:
weird to start with watch. I vote for this order: mobile, tablet, desktop, watch, special